### PR TITLE
Migrate playground to swift3 and new APIs

### LINKED ISSUE #240: Upgrade Swift Playground to support Xcode 8
For some reasons, the Swift Playground doesn't work anymore 😢 even for beta 2.

Relate to #221

### PR DESCRIPTION
**Pull request by tbaranes**
Close #240 
Everything works as expected

### CODE REVIEW

File: IBAnimatable.playground/Pages/Animation Properties.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -15,7 +15,7 @@ let view = CircleView()
 iPhoneView.addSubview(view)
 
 //: animationType: all supported predefined animations can be found in `enum AnimationType`
-view.animationType = String(AnimationType.squeezeInLeft)
+view.animationType = AnimationType.squeeze(way: .in, from: .left)
```
Comment (JakeLin) [inline IBAnimatable.playground/Pages/Animation Properties.xcplaygroundpage/Contents.swift]: Do we need AnimationType here?

File: IBAnimatable.playground/Pages/Chaining Animations.xcplaygroundpage/Contents.swift [hunk 2]
```diff
@@ -15,13 +15,27 @@ let view = CircleView()
...
+                    view.flip(axis: .x) {
+                        view.flip(axis: .y) {
+                            view.slideFade(.out, direction: .down)
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (JakeLin) [inline IBAnimatable.playground/Pages/Chaining Animations.xcplaygroundpage/Contents.swift:32]: It looks horrible now, off cause not your fault, I think the next big thing can be trainable animation methods.

File: IBAnimatable.playground/Pages/Predefined Animations.xcplaygroundpage/Contents.swift [hunk 3]
```diff
@@ -11,66 +11,62 @@ PlaygroundPage.current.liveView = iPhoneView
...
+//view.flip(axis: .y)
+//view.morph(repeatCount: 1)
+//view.squeeze(repeatCount: 1)
+//view.flash(repeatCount: 1)
+//view.wobble(repeatCount: 1)
+//view.swing(repeatCount: 1)
+//view.rotate(direction: .ccw, repeatCount: 10)
+//view.rotate(direction: .cw, repeatCount: 10)
+//view.moveTo(x: 100, y: 100)
+//view.moveBy(x: 100, y: 100)
```
Comment (JakeLin) [inline IBAnimatable.playground/Pages/Predefined Animations.xcplaygroundpage/Contents.swift:70]: Great, they will be in migration guide.

**Review by JakeLin — APPROVED** [review]
Just some small comments on having AnimationType in front of the enum, otherwise all good 👍👍👍💥💥💥

File: IBAnimatable.playground/Pages/Chaining Animations.xcplaygroundpage/Contents.swift (hunk 2 above)
Comment (tbaranes) [reply IBAnimatable.playground/Pages/Chaining Animations.xcplaygroundpage/Contents.swift:32]: Agree, it would be nice to improve it 👍

**Pull request merged**